internal/cmd/grow_container.go

```diff
@@ -38,6 +41,7 @@ with its identifier (e.g. disk1 or /dev/disk1). The string
 	growArgs := growContainer{}
 	cmd.PersistentFlags().StringVar(&growArgs.id, "id", "", `container identifier to be resized or "root"`)
 	cmd.PersistentFlags().BoolVar(&growArgs.dryrun, "dry-run", false, "run command without mutating changes")
+	cmd.PersistentFlags().DurationVar(&growArgs.timeout, "timeout", 5*time.Minute, "Set the timeout for the command (e.g. 30s, 1m, 1.5h)")
```

Comment:
Suggestion: this magic number is pretty important to us - I suggest moving to a constant with a comment why this number is our default.